src/gsi/m_berror_stats.f90

```diff
@@ -646,7 +871,7 @@ subroutine setcoroz_(coroz,mype)
          enddo
       enddo
    enddo
-   work_oz(nsig+1,mm1)=real(lon1*lat1,r_kind)
+   work_oz(nsig+1,mm1)=float(lon1*lat1)
```

Comment:
why float and not real + rkind? I assume r_kind is float?

Reply:
Revert back to `develop` to use `real` and `r_kind`.  Make the same change on line 865.  Done at 68d52ac.